modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js

```diff
@@ -88,16 +90,48 @@ function FieldBase({
 		return languageValues;
 	}, [localizedValue, editingLanguageId, name]);
 	const repeatedIndex = useMemo(() => getRepeatedIndex(name), [name]);
+	const hasLabel = (label && showLabel) || required || tooltip || repeatable;
+	const hasError = displayErrors && errorMessage && !valid;
+	const fieldDetailsId = name + '_fieldDetails';
+	let fieldDetails = '';
+	let parentDivTabIndex;
+	let parentDivAriaLabelledby;
```

Comment:
@rodrigopaulino, please sort alphabetically the variables as much as possible. Thanks.